eng/pipelines/migration.yml

```diff
@@ -0,0 +1,118 @@
+parameters:
+- name: Repos
+  type: object
+  default:
+    # The exclude/include paths are valid for ubuntu agents. If use windows, please follow the format like "^.\\sdk\\" 
+    - RepoOwner: Azure
+      RepoName: azure-sdk-tools
+      ExcludePaths: ''
+      IncludePaths: ''
+    - RepoOwner: Azure
+      RepoName: azure-sdk
+      ExcludePaths: "^./eng/common/*"
+      IncludePaths: ''
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-android
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/resources/session-records/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-c
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-cpp
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-go
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-ios
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-java
+      ExcludePaths: "^./sdk/|^./eng/common/|/resources/session-records/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-js
+      ExcludePaths: "^./sdk/|^./eng/common/|/recordings/node/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-net
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/SessionRecords/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-python
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/recordings/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+# The input parameter format is like:
```

Comment:
We might want to set this as the default so folks know the format and can just tweak as needed.

Reply:
It is default in pipeline UI.
![image](https://user-images.githubusercontent.com/48036328/210665076-add1d8cb-07e0-4c0c-8509-a023e9d35946.png)